### ISSUE TITLE
No softmax before categorical loss?

### ISSUE THREAD

**Issue by hopl1t** [ikostrikov/pytorch-a2c-ppo-acktr-gail#282]
Hi
Thanks so much for sharing this, what a great repo.
I've noticed that the final actor layer is not really activated, rather a distribution object (say categorical) is used.
Later the log probabilities are taken to compute the actor's loss.
Don't we lose the desired mesh that the softmax function gives us in this case?
IE we encourage good actions and discourage bad actions  less then if we'd used softmax, right?
Just wanted to ask is this on propose or did I misunderstand the code?

Thanks!
